.gitignore

```diff
@@ -1,4 +1,3 @@
 target/
-Cargo.lock
```

Comment:
This seems to be standard practice for workspaces with both library and binary crates.